scripts/v3-parity.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+export CHROME_HEADLESS=true
+setup_service google-chrome-stable 83.0.4103.61-1
+
+source $OKTA_HOME/$REPO/scripts/setup.sh
+
+export TEST_SUITE_TYPE="junit"
+export TEST_RESULT_FILE_DIR="${REPO}/build2/reports/junit"
+echo $TEST_SUITE_TYPE > $TEST_SUITE_TYPE_FILE
+echo $TEST_RESULT_FILE_DIR > $TEST_RESULT_FILE_DIR_FILE
+
+# target specs enabled in v3
+ENABLED_TESTS=$(cat <<-END
+test/testcafe/spec/Smoke_spec.js
```

Comment:
how about just another .testcaferc.js? I am pretty sure fixtures can be filtered there, so devs can run all parity tests locally at once

Reply:
https://testcafe.io/documentation/402638/reference/configuration-file#filterfixture

Reply:
Cool. That looks better than this list in sh